### PR TITLE
Ansible update

### PR DESCRIPTION
**Pull request by ayman-m**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
I added a new command to query job events by Id.

## Minimum version of Cortex XSOAR
- [ x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ayman-m** [comment]
@bziser with regards to the  "Possible backwards compatibility break" validation error, I did change the command !ansible-tower-job-events-list removing the "id" argument as the id is not a parameter that can be used with the API endpoint "/api/v2/job_events" as per Ansible's documentation and my testing of this command, the command doesn't return any job event data if the "id" argument is.

The id parameter can be used with a different API endpoint "/api/v2/jobs/{job_id}/job_events/" that's why I added an additional command to use the correct API function.

**Comment by bziser** [comment]
> @bziser with regards to the "Possible backwards compatibility break" validation error, I did change the command !ansible-tower-job-events-list removing the "id" argument as the id is not a parameter that can be used with the API endpoint "/api/v2/job_events" as per Ansible's documentation and my testing of this command, the command doesn't return any job event data if the "id" argument is.
> 
> The id parameter can be used with a different API endpoint "/api/v2/jobs/{job_id}/job_events/" that's why I added an additional command to use the correct API function.

As we discussed on slack, the better approach will be to keep the id argument and pass it to the url_suffix.

File: Packs/AnsibleTower/Integrations/AnsibleTower/AnsibleTower.py [hunk 1]
```diff
@@ -365,6 +365,31 @@ def list_job_events(client: Client, args: dict) -> CommandResults:
     )
 
 
+def list_job_events_by_id(client: Client, args: dict) -> CommandResults:
```
Comment (bziser) [inline Packs/AnsibleTower/Integrations/AnsibleTower/AnsibleTower.py:368]: Please add unit test for the new command.

**Comment by bziser** [comment]
@ayman-m 
To maintain compatibility, I suggest keeping the old command name: `ansible-tower-job-events-list`.
Your new command should be named: `ansible-tower-job-events-list-by-job-id`.
Please update your pull request with these changes.
Thanks for contributing!

**Review by bziser — APPROVED** [review]

**Pull request merged**